docs/specs/devcontainer-features-distribution.md

```diff
@@ -184,3 +184,7 @@ An example project is illustrated below:
 
 }
 ```
+
+### Deprecating
+
+To deprecate an existing Feature, move the Feature's [source code](#source-code) out of the `src` folder or delete the source code. This will ensure that the Feature's metadata is not added to the [devcontainer-collection.json](#devcontainer-collectionjson). The Dev Container index used by the [supporting tools](./supporting-tools.md) is built daily, hence, the deprecated Feature will vanish from the list.
```

Comment:
This isn't quite "deprecating", more "orphaning" the Feature and not supplying updates.  We (or a supporting tool) would have no idea that the user isn't planning on updating the Feature anymore (a supporting tool would not have the information it needs to display a deprecation warning, for example).

I think the concept of "deprecation" is more tied to the aliasing property (https://github.com/devcontainers/spec/issues/146) you're looking into.